include/linux/kfence.h

```diff
@@ -40,7 +40,7 @@ extern struct static_key_false kfence_allocation_key;
 static __always_inline bool is_kfence_address(const void *addr)
 {
 	return unlikely((char *)addr >= __kfence_pool &&
```

Comment:
Doesn't this check guarantee that `addr` is non-null?

Reply:
Jann's concern was that if __kfence_pool's pointer is NULL because KFENCE was never initialized.